ocaml/idl/datamodel.ml

```diff
@@ -8649,10 +8649,59 @@ let vgpu_type =
 		]
 	()
 
-(** PVS_farm
+(** PVS
  * http://www.uk.xensource.com/~robho/pvs-direct/#external-interfaces-xenapi-pvs_farm-new-class
  *)
 
+module PVS_Farm = struct
+  let lifecycle = [Published, rel_dundee_plus, ""]
+  let refset x  = Set(Ref x)
```

Comment:
Since this is a generally useful helper function, you could put it in datamodel_types.ml (in the `Types` module perhaps).